specs/core/0_beacon-chain.md

```diff
@@ -65,7 +65,7 @@
             - [`is_slashable_validator`](#is_slashable_validator)
             - [`is_slashable_attestation_data`](#is_slashable_attestation_data)
             - [`is_valid_merkle_branch`](#is_valid_merkle_branch)
-        - [Misc](#misc)
+        - [Misc](#misc-1)
```

Comment:
I like this resolution of duplicate headers. Can you do `#### Helper functions` and `#### Rewards and penalties` too please?

Reply:
Also `### Attestations` in the V-guide.

Reply:
I think they are already in this format, aren't they? :)

Reply:
Oh, they are. My bad. 🤦‍♂